libmamba/include/mamba/core/transaction.hpp

```diff
@@ -107,6 +107,9 @@ namespace mamba
         MultiPackageCache& package_caches,
         std::vector<detail::other_pkg_mgr_spec>& other_specs
     );
+
+    // NOTE: This can be moved to somewhere else if more appropriate
```

Comment:
Maybe to a place specific to output/console stream? (to be defined after the planned work/refactoring on the output/logs)

Reply:
How about mentioning https://github.com/mamba-org/mamba/issues/2288?
```suggestion
    // NOTE: This can be moved to somewhere else if more appropriate
    // See: https://github.com/mamba-org/mamba/issues/2288
```